samples/client/petstore/php/SwaggerClient-php/lib/ObjectSerializer.php

```diff
@@ -162,6 +162,37 @@ public function toString($value)
     }
 
     /**
+     * Serialize an array to a string.
+     *
+     * @param array  $collection       collection to serialize to a string
+     * @param string $collectionFormat the format use for serialization (csv,
+     * ssv, tsv, pipes, multi)
+     *
+     * @return string
+     */
+    public function serializeCollection(array $collection, $collectionFormat, $allowCollectionFormatMulti=false)
+    {
+        if ($allowCollectionFormatMulti && ('multi' === $collectionFormat)) {
+            return http_build_query($collection, null, '&');
```

Comment:
I think this does not handle the "multi" collectionFormat correctly. Assuming a parameter "foo" has an array value of 2 elements, "bar" and "baz", in the `$collection` here, then this is expected to be included in the URL:
`foo=bar&foo=baz`
However, the implementation of `serializeCollection` seems to return string `0=bar&1=baz`, which will be included into the URL via [ApiClient.php](https://github.com/arnested/swagger-codegen/blob/collectionFormat/samples/client/petstore/php/SwaggerClient-php/lib/ApiClient.php#L175) and the URL would contains:
`foo=0=bar&amp;1=baz`